src/briefcase/commands/base.py

```diff
@@ -546,9 +522,8 @@ def clone_options(self, command):
 
         :param command: The command whose options are to be cloned
         """
-        self.input.enabled = command.input.enabled
-        self.logger = command.logger
         self.is_clone = True
+        self.save_log = command.save_log
```

Comment:
Should these properties be passed in as `**kwargs` to the constructor, rather than "fixed" after the construction of the main command object?

Reply:
Moving to keyword arguments for `BaseCommand`.

Reply:
Actually, I moved `save_log` to `Log` so it's handled similarly to `verbosity` and `is_clone` is now set by the consolidated command factory.